### PR TITLE
Updated SABnzbd Testing to 1.0.3

### PR DESCRIPTION
**Pull request by KaraokeStu**
Updated SABnzbd Testing to 1.0.3

There is not really any official testing code from the SABnzbd guys at present, but it makes sense to keep this in line with the released version

### CODE REVIEW

File: spk/sabnzbd-testing/Makefile [hunk 1]
```diff
@@ -15,7 +15,34 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 ADMIN_PORT = 8089
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd Testing
-CHANGELOG = "Update to 1.0.0"
+CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost.
```
Comment (Dr-Bean) [inline spk/sabnzbd-testing/Makefile]: As per the comment in the other PR: this should be updated to say `Update to 1.0.3`. Let's leave the warning in for now.
The rest of the additions should be removed: Package Center won't be able to handle it.

**Comment by Dr-Bean** [comment]
Other than the one comment, LGTM. :+1:

**Comment by KaraokeStu** [comment]
Updated commit with dependency on unzip and made changelog much shorter with HTML line breaks (as it had before)

**Comment by KaraokeStu** [comment]
Updated to use correct HTML and remove cumulative changelog.
Made the warning red and italic ;)

**Comment by KaraokeStu** [comment]
Updated as per @Dr-Bean requested

**Comment by Dr-Bean** [comment]
We might not merge this after all. SABnzbd1.1.0Beta1 has been released yesterday.

I just checked it out, but there might be an issue with it: the first step in the sabnzbd install wizard (not our package wizard) doesn't seem to work because the languages don't show up. It continues fine after that though. I don't think it's a local issue with my browser or anything, but I haven't checked the code to see if changes were made to that part or anything.

@KaraokeStu care to check for yourself? Note that if you end up opening a PR, there's been an icon change which should be included as well.

**Comment by KaraokeStu** [comment]
Hi,

No problem I'll check it out.

I'll keep the testing package in line with the beta tree.

**Comment by Dr-Bean** [comment]
Opened issue https://github.com/sabnzbd/sabnzbd/issues/593 for the missing languages.

**Comment by KaraokeStu** [comment]
Closing pull request to open a new one for SABnzbd 1.1.0 Beta 1

**Pull request closed without merging**